pol-core/pol/multi/customhouses.h

```diff
@@ -175,6 +176,8 @@ void CustomHousesSendFull( UHouse* house, Network::Client* client,
 void CustomHousesSendFullToInRange( UHouse* house, int design, int range );
 void CustomHousesSendShort( UHouse* house, Network::Client* client );
 void CustomHouseStopEditing( Mobile::Character* chr, UHouse* house, bool send_pkts = true );
+void CustomHouseStopEditing( Mobile::Character* chr, UHouse* house, ItemList& foo,
```

Comment:
foo? :)
A comment why these two versions exists would be good when we cannot get rid of the second version, otherwise we will forget it.